internal/testing/releasehelper.go

```diff
@@ -39,16 +39,27 @@ The tool processes all modules listed in the 'allmodules'
 file. For each module it handles all dependencies on
 other gocloud.dev modules.
 
--replace adds 'replace' directives to point to local versions
-for testing. -dropreplace removes these directives.
+Run it from the root directory of the project, as follows:
 
-Run it from the root directory of the project.
+$ %s <command>
+
+Where command is one of the following:
+
+  addreplace    adds 'replace' directives to point to local versions
+                for testing.
 
-Usage:
+  dropreplace   removes these directives.
+
+  help          prints this usage message
+
+Run it from the root directory of the project.
 `
 
-var doReplace = flag.Bool("replace", false, "add replace directives")
-var doDropReplace = flag.Bool("dropreplace", false, "drop replace directives")
+func printHelp() {
+	_, binName := filepath.Split(os.Args[0])
```

Comment:
Sorry, what does this do? Why not just `fmt.Println(helpText)`? And maybe `fmt.Fprintf(os.Stderr` ?

Reply:
Without this the command is shown as `/tmp/go-build736844560/b001/exe/releasehelper` when running with `go run`.

Just figured it's nice to have the command name there for usage.

Reply:
Ah, I see, I missed that `helpText` had a '%s' in it.